src/api/contract/getWithdrawalBalanceEthAddress.js

```diff
@@ -0,0 +1,30 @@
+const DVFError = require('../../lib/dvf/DVFError')
+const BN = require('bignumber.js')
+
+module.exports = async (dvf, token, address) => {
+  address = address || dvf.get('account')
```

Comment:
Do we want to support no address as input for a particular use case ?

Reply:
Is there a reason to not default to current user?

Reply:
It just adds `if` in various parts of the code, possibly leading to ununderstood behavior. I usually prefer when there is one way to call a function unless we want the function to accomplish two different functional things.
But if this is a choice I am fine with it, just wanted to clarify

Reply:
Indeed I agree with reducing branching. In this case I followed the example of the other getWithdrawalBalance which was defaulting to a set stakKey, but its a good point, I'll remove the default address so it has to be passed in

Reply:
Note that removal of the default account in [this commit](https://github.com/DeversiFi/dvf-client-js/pull/303/commits/594774ade98239d62856ec99bf17baf432b4dd90) broke `[examples/getWithdrawals](https://github.com/DeversiFi/dvf-client-js/blob/5fe69d484dd44348e4e2d70a8e15f523e5daa822/examples/15.getWithdrawals.js#L39)`

Reply:
Fair point, we can certainly add it back, however there was a change in starkware's contract when it comes to fetching withdrawals. Addresses can now be either starkKey or ethAddress depending on the type of withdrawal. This change aimed to ensure of the two is explicitly passed it since simply using `ethAddress` will not guarantee that you get all your withdrawals anymore

Reply:
fair point, we can certainly add this back in, it is important to note there are 2 sources for getting withdrawals now, if we're defaulting this one, we should default the other one to user's starkkey as well